src/agent/state/legacy-state.ts

```diff
@@ -40,9 +40,11 @@ export function evaluate(
   frame: v8.FrameMirror
 ): {error: string | null; mirror?: v8.ValueMirror} {
   // First validate the expression to make sure it doesn't mutate state
+  // eslint-disable-next-line @typescript-eslint/no-var-requires
```

Comment:
Instead of this, could you install `@types/acorn` and use that?

Reply:
used `import * as acorn from 'acorn';`
Thanks